pkg/acsengine/defaults-apiserver.go

```diff
@@ -99,6 +99,11 @@ func setAPIServerConfig(cs *api.ContainerService) {
 		}
 	}
 
+	// Pod Security Policy configuration
+	if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnablePodSecurityPolicy) {
+		defaultAPIServerConfig["--admission-control"] = "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,DenyEscalatingExec,AlwaysPullImages,SecurityContextDeny,PodSecurityPolicy"
```

Comment:
To make `--admission-control` more maintainable going forward, let's just add `",PodSecurityPolicy"` to the end of the existing string at this point in the code (after default assignment) if `EnablePodSecurityPolicy` is true, rather than replacing the entire string.

Reply:
updated. I was going to suggest we refactor the admission-control flags into a []string or a map instead of a string.